WebRoot/js/zimbraAdmin/globalconfig/controller/ZaGlobalConfigViewController.js

```diff
@@ -416,6 +416,11 @@ function () {
 					isIPOrHostName = false;
 				}
 
+				// In case of localhost, lets not verify IPV4
+				if (chunks[1] === 'localhost') {
```

Comment:
Why are we getting localhost as IP?

Reply:
I don't know the exact scenario but its being used in zoom setup.

Reply:
After discussion with platform team, i came to know that If the SSDB is installed on local server then localhost is the ideal way to bypass network layer all together.
For the Zoom integration we used redis as a Ephemeral backend and installed it on local mailbox server
and used ssdb:localhost:6379